src/script/calling/entities/FlowEntity.js

```diff
@@ -901,6 +903,7 @@ z.calling.entities.FlowEntity = class FlowEntity {
             obfuscated: [
               transformedSdp.type,
               iceCandidates.length,
+              iceCandidateTypesLog,
```

Comment:
I also fixed the production log to include the types of ice candidates that we send